src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -480,9 +480,13 @@ namespace wallet_rpc
       std::list<transfer_destination> destinations; // Array of destinations to receive LOKI.
       uint32_t account_index;                       // (Optional) Transfer from this account index. (Defaults to 0)
       std::set<uint32_t> subaddr_indices;           // (Optional) Transfer from this set of subaddresses. (Defaults to 0)
-      uint32_t priority;                            // Set a priority for the transaction. Accepted Values are: default (1), or 0-3 for: unimportant, normal, elevated, priority.
-      bool blink;                                   // Specifies that the tx should be blinked (`priority` will be ignored).
+      uint32_t priority;                            // Set a priority for the transaction. Accepted Values are: 0-5 for: default, unimportant, normal, elevated, priority, blink.
+#if LOKI_DEPRECATE_AT_VERSION(7, 1, 0)
+      bool blink;                                   // (Deprecated) Specifies that the tx should be blinked (`priority` will be ignored).
```

Comment:
Is there any reason to not just remove this now, rather than waiting for v7?  In any event, using a macro like this to force code to be removed in a future version rather than just using a comment and as we've been doing up until now seems like serious overkill (especially when it requires undoing the version header macro inclusion location).